File: fcs59/python/move-zeros.py
def move_zeros(nums):
    l = []
    for x in nums:
        if x!=0:
            l.append(x)

    count_zeros = nums.count(0)
    
    return l + [0] * count_zeros

print(move_zeros([1, 2, 0, 5, 0, 11, 3, 0, 56]))

File: fcs59/js/move-zeros.js
function move_zeros(nums) {
  let l = [];

  for (let x of nums) {
    if (x !== 0) {
      l.push(x);
    }
  }

  let count_zeros = nums.length - l.length;

  for (let i = 0; i < count_zeros; i++) {
    l.push(0);
  }
  return l;
}

console.log(move_zeros([1, 2, 0, 5, 0, 11, 3, 0, 56]))